### PR TITLE
63 read remote

### PR DESCRIPTION
**Pull request by jules32**
Hi @betolink, would you please review? Goes with #63

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,128 +3,6 @@
 This Earthdata Cloud Cookbook is being developed by the [NASA-Openscapes team](https://nasa-openscapes.github.io/).
 
 ### How to contribute to our book
-Information for our team [to contribute to the Cookbook](https://nasa-openscapes.github.io/earthdata-cloud-cookbook/contributing/).
+Information for our team [to contribute to the Cookbook](https://nasa-openscapes.github.io/earthdata-cloud-cookbook/contributing/). This includes setup and workflow instructions.
```
Comment (betolink) [inline README.md:6]: Nice, I think this is a good approach to not overwhelm users when they open the repo.

**Comment by betolink** [comment]
Nicely done, with this approach we could target stand alone notebooks and include them in the book. If they are rendered I think.

**Review by betolink — APPROVED** [review]
sorry for the delay, good PR! looking forward to try it with one of the NSIDC tutorials notebooks.

**Comment by jules32** [comment]
Merging this following @betolink's review and approval

**Pull request merged**